content/chapter-03/SystemMetricsLab.md

```diff
@@ -1 +1,40 @@
-# Lab: common tools for system metrics
\ No newline at end of file
+# Lab: common tools for system metrics
+
+* In this lab, two Virtual Machine instances will be created: One will run a stress test, and the other will run the application to be tested.
+
+* Tester instance
+  * Create a small instance to launch a stress test tool. e.g.: AWS EC2 t2.micro, AMI: Ubuntu Server 16.04 LTS (HVM), SSD Volume Type - ami-66506c1c
+  * Configure that instace to have 8GB of storage.
+  * ssh to the instance and install [Apache Bench](https://en.wikipedia.org/wiki/ApacheBench):
+    * `sudo apt-get update`
+    * `sudo apt-get install apache2-utils`
+  * Validate the installation with `man ab`
+  * Run a first test to Wikipedia: `ab -n 100 -c 10 "http://en.wikipedia.org/wiki/Main_Page"`
+
+
+* Application instance
+  * Create another small install to install the `get_a_book` application and monitor it
```

Comment:
Create another small instance to install `get_a_book` application and to monitor it

Reply:
Done